Tools/MonoGame.Content.Builder/version/Version.props

```diff
@@ -0,0 +1,5 @@
+﻿<Project>
+  <PropertyGroup>
+    <MonoGameContentBuilderVersion>$([System.IO.Path]::GetFileName($([System.IO.Path]::GetDirectoryName($(MSBuildThisFileFullPath)))))</MonoGameContentBuilderVersion>
```

Comment:
`$([System.IO.Path]::GetDirectoryName($(MSBuildThisFileFullPath)))` => You can use $(MSBuildThisFileDirectory).

Reply:
That's what I tried first, but `$(MSBuildThisFileDirectory)` has a trailing backslash, which unfortunately means `GetFileName()` won't work as you expect unless you also trim the trailing separator. [This answer](https://stackoverflow.com/a/57691321) and some of the comments on this post discuss this. What I have now is the most robust solution I could find.